presto-main/src/main/java/io/prestosql/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -370,7 +374,14 @@ public Void visitJoin(JoinNode node, Void context)
             for (JoinNode.EquiJoinClause clause : node.getCriteria()) {
                 joinExpressions.add(clause.toExpression());
             }
-            node.getFilter().ifPresent(joinExpressions::add);
+
+            Optional<Expression> filter = node.getFilter();
+            List<DynamicFilters.Descriptor> dynamicFilters = ImmutableList.of();
```

Comment:
I think dynamic filter assignments should never exist in join after predicate pushdown therefore I wouldn't change the code here

Reply:
Removed these changes as dynamicFilters shouldn't exist in JoinNode after predicate pushdown